src/types.rs

```diff
@@ -187,10 +187,6 @@ impl StaticType for Vec<String> {
     }
 }
 
-pub trait InstanceType {
```

Comment:
This breaks API in theory, depends on if you want to ignore that. Nobody probably uses this

Reply:
We'll keep this under silence if you don't mind. I want to wait a bit more for a bigger release. ;)

Reply:
Fine with me, but it should be a conscious decision and not an accident

Reply:
I have to admit it completely slipped out of my review. Should have been more careful...